### ISSUE TITLE
Mobile Editor

### ISSUE THREAD

**Issue by ap0l0** [paperbits/paperbits-demo#187]
Any plans on making it easier to edit or work the editor from a mobile/touch device?

**Comment by azaslonov** [comment]
Yes, there is a bunch of work items to enable better touch support and proper editor layouts for mobile screens.

**Comment by ap0l0** [comment]
Thanks @azaslonov , we are looking to an editor able to handle edition from Mobile and so far nothing is well prepared for that. Only GrapeJS has a "touch plugin" but doesn't help much. Is there anything now we can use from your library to be able to easily edit from a Mobile device?

Thanks!

**Comment by azaslonov** [comment]
This probably depends on what you are looking for. Most of the editing functionality available even now, maybe not very convenient, but it's there. Do you mind expanding on your scenarios a little bit more?

**Comment by ap0l0** [comment]
I would say:

1. enabling touch features such as drag drop
2. resizing an element (image, buttons, etc.)
3.  Simplified UI view for mobile edits, for example if I touch an element then the options to edit that element will appear

**Comment by ap0l0** [comment]
https://user-images.githubusercontent.com/14894967/124364766-a6725e80-dc11-11eb-94c8-7415909fd480.mov

A quick demo of trying to using it from mobile (Safari iOS 15)

**Comment by azaslonov** [comment]
I see.
> 1. enabling touch features such as drag drop

Yep, drag and drop, not supported yet.

> 2. resizing an element (image, buttons, etc.)

Resizing can be done in the widget editor even now.

> 3. Simplified UI view for mobile edits, for example if I touch an element then the options to edit that element will appear

This is working even now, but editors don't place themselves mobile-friendly way.

As you can see, there are no fundamental issues. We'll be working on those minor issues later this year. Also, contributions are always welcomed :)
